monailabel/tasks/train/bundle.py

```diff
@@ -125,6 +125,9 @@ def _fetch_datalist(self, request, datastore: Datastore):
 
     def _partition_datalist(self, datalist, request, shuffle=False):
         if "detection" in request.get("model"):
+            settings.MONAI_LABEL_TRACKING_ENABLED = (
```

Comment:
you don't have to override the settings.. instead you override the `request["tracking"] = "None"` or even empty or None will be fine too..

Reply:
Thanks, made changes according to the prior PR on disabling mlflow for bundles. All bundles works well.